.github/workflows/ui-tests.yml

```diff
@@ -0,0 +1,74 @@
+---
+name: UI Tests
+on:
+  pull_request:
+
+jobs:
+  test:
+    name: Run UI tests
+    runs-on: ubuntu-20.04
+    timeout-minutes: 25
+
+    steps:
+      - name: Checkout Grafana code
+        uses: percona-platform/checkout@v2
+        with:
+          path: ./grafana
+
+      - name: Run with Node 12
+        uses: percona-platform/setup-node@v2
+        with:
+          node-version: 12
+
+      - name: Get Yarn cache directory
+        id: yarn-cache-dir-path
+        working-directory: ./grafana
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Use Yarn cache
+        uses: percona-platform/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-12-${{ hashFiles('**/yarn.lock') }}
```

Comment:
Is that `12` a version? Do we need to hardcode it?

Reply:
You are right, I'll revert and use matrix strategy